nemo/collections/nlp/models/language_modeling/megatron_base_prompt_learning_model.py

```diff
@@ -59,7 +58,9 @@ class MegatronBasePromptLearningModel(MegatronBaseModel, TextGeneration):
 
     def __init__(self, cfg: DictConfig, trainer: Trainer):
         super().__init__(cfg, trainer)
+        self.init_model(cfg, trainer)
```

Comment:
I see here (in the lines above) that you have imports you don't use - maybe remove them in the refactor 4/n ? vscode should be able to identify them

Reply:
There's quite a few in other files as well

Reply:
Agreed! will do!